java/tools/pom.xml

```diff
@@ -58,13 +58,11 @@
       <groupId>org.apache.hadoop</groupId>
       <artifactId>hadoop-common</artifactId>
       <version>${tools.hadoop.version}</version>
-      <scope>compile</scope>
```

Comment:
Hi, @belugabehr and @dongjoon-hyun .
This seems to break ORC tools uber jar again. I'll make a PR to recover it.